src/applications/accredited-representative-portal/accreditation/21a/containers/IntroductionPage.jsx

```diff
@@ -27,9 +27,155 @@ function StartLink({ children, ...props }) {
+            address starting from high school graduation, undergraduate, to
+            post-graduate. You will also need to provide the dates you attended
+            and the degree received (or major).
+          </p>
+        </va-process-list-item>
+        <va-process-list-item header="Professional affiliations">
+          <p>
+            You will need to provide each jurisdiction you are currently a
+            member of good standing. Along with the name, you will also need to
+            provide the date of admission and your membership or registration
+            number.
+          </p>
+        </va-process-list-item>
+        <va-process-list-item header="Background information">
+          <p>
+            Truthfulness and candor are essential elements of good moral
+            character and reputation relevant to practice before VA. In order to
+            evaluate your character and reputation, you will need to provide
+            details on any convictions, imprisonments, sentences, terminations
+            due to unethical or unlawful behavior, and related topics.
+          </p>
+        </va-process-list-item>
+        <va-process-list-item header="Character references">
+          <p>
+            You will need to provide 3-4 character references who have personal
+            knowledge of your character and qualification to serve as a claims
+            agent or attorney. These references cannot be immediate family
+            members.
+          </p>
+        </va-process-list-item>
+        <va-process-list-item header="Optional supplementary statements">
+          <p>
+            Provide any additional explanations for your answers in previous
+            sections and additional information about why you are applying.
+            These questions are optional.
+          </p>
+        </va-process-list-item>
+        <va-process-list-item header="Review application">
+          <p>Review your answers before submitting.</p>
+        </va-process-list-item>
+      </va-process-list>
+      <SaveInProgressIntro
+        headingLevel={2}
+        prefillEnabled={formConfig.prefillEnabled}
+        messages={formConfig.savedFormMessages}
+        pageList={pageList}
+        startText="Start your Application"
+        customLink={StartLink}
+        hideUnauthedStartLink
+        displayNonVeteranMessaging
+      />
+      <va-omb-info
+        res-burden={45}
+        omb-number="2900-0605"
+        exp-date="3/31/2022"
+      />
+    </article>
+  );
+};
+
+IntroductionPage.propTypes = {
+  route: PropTypes.shape({
+    formConfig: PropTypes.shape({
+      prefillEnabled: PropTypes.bool,
+      savedFormMessages: PropTypes.object,
+      title: PropTypes.string,
+      subTitle: PropTypes.string,
+    }),
+    pageList: PropTypes.arrayOf(PropTypes.object),
+  }),
+  startLink: PropTypes.oneOfType([
+    PropTypes.element,
+    PropTypes.elementType,
+    PropTypes.func,
+  ]),
+};
```

Comment:
These changes seem to already be present in `main`:
https://github.com/department-of-veterans-affairs/vets-website/blob/986d8eeacb6f4bcc318f662a52a9e78eb2264204/src/applications/accreditation/21a/containers/IntroductionPage.jsx

I imagine the diffs will disappear once we merge `main` into the base branch?